java/client/src/main/java/com/youtube/vitess/client/cursor/Row.java

```diff
@@ -29,6 +29,7 @@
   private FieldMap fieldMap;
   private List<ByteString> values;
   private Query.Row rawRow;
+  private boolean lastGetWasNull;
```

Comment:
Can you add a prominent comment that the caller must ensure thread-safety by proper locking when calling "wasNull"? Or use the GetObject() alternative?